src/main/java/ch/njol/skript/registrations/EventValues.java

```diff
@@ -89,13 +106,35 @@ public Class<? extends E>[] getExcludes() {
 		}
 		
 		/**
-		 * Get the error message used when encountering an exclude value.
-		 * @return The error message to use when encountering an exclude
+		 * Get the error message used when encountering an excluded event.
+		 * @return The error message to use when encountering an excluded event
 		 */
 		@Nullable
 		public String getExcludeErrorMessage() {
 			return excludeErrorMessage;
 		}
+
+		/**
+		 * Get the time state of this event value.
+		 * @return Either {@link #TIME_PAST}, {@link #TIME_NOW} or {@link #TIME_FUTURE}
+		 */
+		public int getTime() {
+			return time;
+		}
+
+		/**
+		 * Whether the provided event is one of the excluded events of this {@link EventValueInfo}
+		 * @return Either {@link #TIME_PAST}, {@link #TIME_NOW} or {@link #TIME_FUTURE}
+		 */
+		public boolean isEventExcluded(Class<? extends Event> c) {
+			if (excludes != null)
+				for (Class<? extends Event> exclude : excludes) {
+					if (exclude.isAssignableFrom(c))
+						return true;
+				}
```

Comment:
Java won't run the for loop if null
```suggestion
			for (Class<? extends Event> exclude : excludes) {
				if (exclude.isAssignableFrom(c))
					return true;
			}
```